pyproject.toml

```diff
@@ -14,14 +14,24 @@ license = { file = "licenses/LICENSE.rst" }
 authors = [
     { name = "The Astropy Developers", email = "astropy.team@gmail.com" }
 ]
-dependencies = []
+dependencies = [
+  "array-api-compat",
+  "astropy",
+  "numpy>=2.0",
+]
 dynamic = ["version"]
 
 [project.optional-dependencies]
+# Include other array types than numpy/array-api-strict. Mostly for tests.
+all = [
+    "dask",
```

Comment:
```suggestion
    "dask>=2024.11.2",
```

Reply:
Are you suggesting to simply put things at the latest version, so we don't have to worry about earlier versions?

Reply:
I mean we should set this based on NEP-29, but yes... I was just thinking about easy development.

Reply:
OK, makes sense to me.